### ISSUE TITLE
touch.facebook.com: ads

### ISSUE THREAD

**Issue by WukZarathustra** [uBlockOrigin/uAssets#17607]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://touch.facebook.com/`

### Category

ads

### Description

When loading the page for the first time, advertising comes through on Android. Only when I swipe down and load the page a 2nd time, it blocks the ads. The problem also affects m.facebook.com. 

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>
![Screenshot_2023-04-15-16-01-00-187_com kiwibrowser browser](https://user-images.githubusercontent.com/125025357/232229001-810acee1-a81e-4aa3-96c8-672a2502ca55.jpg)

### Configuration

<details>

```yaml
uBlock Origin: 1.48.4
Chromium Mobile: 111
filterset (summary): 
  network: 233182
  cosmetic: 181552
  scriptlet: 40767
  html: 0
listset (total-discarded, last-updated): 
  added: 
    https://filters.adtidy.org/extension/ublock/filters/3.txt: 38614-3384, 9h.1m
    https://filters.adtidy.org/extension/ublock/filters/6.txt: 13049-627, 8h.59m
    https://raw.githubusercontent.com/ansul1001/Ansul-s-Ad-Filters/master/Ansul's%20Ad%20Filters: 35-0, 8h.25m
    https://winhelp2002.mvps.org/hosts.txt: 8729-290, 2d.4h.42m
    adguard-annoyance: 67268-1873, 9h.5m
    adguard-generic: 69880-1757, 9h.4m
    adguard-social: 20261-1982, 9h.3m
    adguard-spyware-url: 976-2, 9h.2m
    block-lan: 47-0, 4d.5h.15m
    curben-phishing: 69201-2, 7h.29m
    curben-pup: 199-0, 7h.31m
    dpollock-0: 11354-1434, 1h.42m
    fanboy-annoyance: 80612-39101, 5d.16h.38m
    fanboy-cookiemonster: 46546-46546, 5d.16h.37m
    fanboy-social: 20672-20664, 1d.7h.45m
    fanboy-thirdparty_social: 68-20, 5d.16h.36m
    ublock-annoyances: 4994-278, 8h.58m
  default: 
    user-filters: 37-1, never
    DEU-0: 6770-6767, 7h.30m
    adguard-mobile: 8767-131, 8h.39m
    easylist: 64669-18017, 9h
    easyprivacy: 30810-12253, 2d.15h.22m
    plowe-0: 3750-1587, 7d.3h.35m
    ublock-abuse: 78-0, 7d.12h.15m
    ublock-badware: 6604-368, 8h.57m
    ublock-filters: 33937-4440, 8h.55m
    ublock-privacy: 302-68, 7d.11h.51m
    ublock-quick-fixes: 367-233, 7h.33m
    ublock-unbreak: 1981-135, 8h.40m
    urlhaus-1: 7236-3, 7h.32m
filterset (user): [array of 30 redacted]
trustedset: 
  added: [array of 16 redacted]
  removed: 
    chrome-scheme
switchRuleset: 
  added: [array of 469 redacted]
hostRuleset: 
  added: [array of 2046 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  webrtcIPAddressHidden: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 3153 ms (selfie)
  maxAssetCacheWait: 449 ms
popupPanel: 
  blocked: 0
```
</details>

**Comment by iam-py-test** [comment]
1. You are using Kiwi browser, which is known to cause problems and is rarely updated
2. If my memory serves me correctly, only Firefox Mobile is supported on Android
3. You have many non-default lists enabled. [Reset uBo to default settings](https://github.com/gorhill/uBlock/wiki/Dashboard:-Settings#backuprestore-section) and update your lists

**Comment by WukZarathustra** [comment]
Yes in Kiwi browser some addons work more. But if the browser is updated so rarely, it's not even that good to use it. 

However, it also affects the Firefox browser with exactly the same problem. I'll have a look at the additional filters. And see how I set them to default.

**Comment by iam-py-test** [comment]
All Facebook issues need to be reported at https://github.com/uBlockOrigin/uAssets/issues/3367
